### PR TITLE
Add new security workflows

### PR DESCRIPTION
**Pull request by mattops**

### CODE REVIEW

**Review by psoleckimoj — APPROVED** [review]
LGTM!

File: rename-project.bash [hunk 1]
```diff
@@ -76,9 +76,13 @@ sed -i -z -E \
   -e "s/security:\n    triggers:\n      - schedule:\n          cron: \"30 5/security:\n    triggers:\n      - schedule:\n          cron: \"$RANDOM_MINUTE $RANDOM_HOUR/" \
```
Comment (petergphillips) [inline rename-project.bash:76]: targetting wrong file

File: .github/workflows/security_npm_dependency.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+name: Security npm dependency check
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "19 6 * * MON-FRI" # Every weekday
+jobs:
+  security-npm-dependency-check:
+    name: Project security npm dependency check
+    uses: ministryofjustice/hmpps-github-actions/.github/workflows/security_npm_dependency.yml@v0.7.1 # WORKFLOW_VERSION
```
Comment (petergphillips) [inline .github/workflows/security_npm_dependency.yml:9]: don't you want to float on minor version - why pin to patch version?  Maybe better to release `v1` and then float on major version instead

**Review by petergphillips — APPROVED** [review]
LGTM

**Pull request merged**